packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -174,124 +175,137 @@ async function startContainerProvider(
 </script>
 
 <SettingsPage title="Resources">
-  <span slot="subtitle">
+  <span slot="subtitle" class="{providers.length > 0 ? '' : 'hidden'}">
     Additional provider information is available under <a
       href="/preferences/extensions"
       class="text-gray-400 underline underline-offset-2">Extensions</a>
   </span>
   <div>
-    {#each $providerInfos as provider}
-      <div class="bg-zinc-800 mt-5 rounded-md p-3 divide-x divide-gray-600 flex">
-        <div>
-          <!-- left col - provider icon/name + "create new" button -->
-          <div class="min-w-[150px] max-w-[200px]">
-            <div class="flex">
-              {#if provider.images.icon}
-                {#if typeof provider.images.icon === 'string'}
-                  <img src="{provider.images.icon}" alt="{provider.name}" class="max-w-[40px] h-full" />
-                  <!-- TODO check theme used for image, now use dark by default -->
-                {:else}
-                  <img src="{provider.images.icon.dark}" alt="{provider.name}" class="max-w-[40px]" />
-                {/if}
-              {/if}
-              <span class="my-auto text-gray-300 ml-3 break-words">{provider.name}</span>
-            </div>
-            <div class="text-center mt-10">
-              {#if provider.containerProviderConnectionCreation || provider.kubernetesProviderConnectionCreation}
-                {@const providerDisplayName =
-                  (provider.containerProviderConnectionCreation
-                    ? provider.containerProviderConnectionCreationDisplayName || undefined
-                    : provider.kubernetesProviderConnectionCreation
-                    ? provider.kubernetesProviderConnectionCreationDisplayName
-                    : undefined) || provider.name}
-                <!-- create new provider button -->
-                <Tooltip tip="Create new {providerDisplayName}" bottom>
-                  <button
-                    class="pf-c-button pf-m-primary"
-                    aria-label="Create new {providerDisplayName}"
-                    type="button"
-                    on:click="{() => router.goto(`/preferences/provider/${provider.internalId}`)}">
-                    Create new ...
-                  </button>
-                </Tooltip>
-              {/if}
-            </div>
+    {#if providers.length === 0}
+      <div class="p-2 flex flex-col bg-zinc-800 rounded-lg mt-4">
+        <div class="flex flex-col items-center text-center mt-10 mb-20">
+          <EngineIcon />
+          <p class="text-lg text-gray-200">No resource found</p>
+          <div class="text-sm text-gray-400 mt-2">
+            <p>Create a container engine provider or install an extension to start</p>
```

Comment:
It's not limited to container engine, can be kubernetes engine as well
Also extension need to be started not only installed

```suggestion
            <p>Start an extension that manage container or Kubernetes engines</p>
```

Reply:
Sentence is just a suggestion, can be anything else that includes 'Kubernetes engine' as well